src/Controls/src/Core/Compatibility/Handlers/ListView/iOS/ListViewRenderer.cs

```diff
@@ -1276,7 +1276,7 @@ public override void Scrolled(UIScrollView scrollView)
 					return;
 
 				var args = new ScrolledEventArgs(scrollView.ContentOffset.X, scrollView.ContentOffset.Y);
-				if (!_list.TryGetTarget(out var list))
+				if (_list.TryGetTarget(out var list))
```

Comment:
Changes applied fixing https://github.com/dotnet/maui/issues/20025
Mistake in comparison.